FeedAPIChallenge/Feed API/RemoteFeedLoader.swift

```diff
@@ -19,6 +19,47 @@ public final class RemoteFeedLoader: FeedLoader {
 	}
 
 	public func load(completion: @escaping (FeedLoader.Result) -> Void) {
-		fatalError("Must be implemented")
+		client.get(from: url) { [weak self] result in
+			guard self != nil else { return }
+			switch result {
+			case .failure:
+				completion(.failure(Error.connectivity))
+			case .success((let data, let httpResponse)):
```

Comment:
This success `case` can be simplified with a single let.
```suggestion
			case let .success((data, httpResponse)):
```